schema/modules/mqModule/RabbitMQModule.go

```diff
@@ -17,21 +17,22 @@ package mqModule
 
 import (
 	"fmt"
+	"os"
+	"reflect"
+	"strconv"
+
 	"github.com/rs/zerolog"
 	"github.com/th2-net/th2-common-go/schema/common"
 	"github.com/th2-net/th2-common-go/schema/factory"
 	"github.com/th2-net/th2-common-go/schema/queue/MQcommon"
 	"github.com/th2-net/th2-common-go/schema/queue/configuration"
 	event "github.com/th2-net/th2-common-go/schema/queue/event/impl"
-	"github.com/th2-net/th2-common-go/schema/queue/message/impl"
-	"os"
-	"reflect"
-	"strconv"
+	message "github.com/th2-net/th2-common-go/schema/queue/message/impl"
 )
 
 const (
-	RABBIT_MQ_CONFIG_FILENAME = "rabbitmq"
-	MQ_ROUTER_CONFIG_FILENAME = "routermq"
+	RABBIT_MQ_CONFIG_FILENAME = "rabbitMQ"
+	MQ_ROUTER_CONFIG_FILENAME = "mq_router"
```

Comment:
Could you please correct names in all modules. You can find correct values by the link https://github.com/th2-net/th2-common-j/blob/master/src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java#L90